zqd/space/filespace.go

```diff
@@ -27,23 +25,28 @@ func (s *fileSpace) Info(ctx context.Context) (api.SpaceInfo, error) {
 	if err != nil {
 		return api.SpaceInfo{}, err
 	}
-	pcapsize, err := s.PcapSize()
-	if err != nil {
-		return api.SpaceInfo{}, err
-	}
 	du := s.conf.DataURI
 	if du.IsZero() {
 		du = s.path
 	}
 
 	si.Name = s.conf.Name
 	si.DataPath = du
-	si.PcapSize = pcapsize
-	si.PcapSupport = s.PcapPath() != ""
-	si.PcapPath = s.PcapPath()
 	return si, nil
 }
 
+func (s *fileSpace) PcapStorage() *pcapstorage.Store {
+	return s.pcapstore
+}
+
+func (s *fileSpace) dataURI() iosrc.URI {
```

Comment:
This function doesn't appear to be used.

Reply:
I forgot that I added this, I will use this.